test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -62,6 +62,16 @@ function RunTestCase
     }
 }
 
+Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @('CI') {
+
+    It "Get-Help Get-Help returns information about the help system." {
+
+        $helpContent = Get-Help Get-Help | Out-String
+        $helpContent | Should Match Get-Help
```

Comment:
my assumption is that this test is not validating the actual formatting of the help, but that we're returning the right thing

---
In reply to: [97399837](https://github.com/PowerShell/PowerShell/pull/3032#discussion_r97399837) [](ancestors = 97399837)